clang/include/clang/CodeGen/ModuleBuilder.h

```diff
@@ -27,6 +27,9 @@ namespace llvm {
   }
 }
 
+// Prefix for __builtin_verbose_trap.
+#define CLANG_VERBOSE_TRAP_PREFIX "__llvm_verbose_trap"
```

Comment:
@ahatanak Nit: Why does the macro name mention `CLANG` but the string mentions `llvm` instead of Clang? Seems like this prefix is specific to Clang and not LLVM.

Also here’s suggestion for a shorter prefix `__llvm_trap_msg`. It’s not a big deal so feel to ignore this advice.

Reply:
Does this have to be a macro or could it be a C++ constant?

Reply:
If we used a C++ constant, wouldn't it be harder to make a compile time constant string (e.g., `^__llvm_verbose_trap: `)?

https://github.com/llvm/llvm-project/pull/80368/files#diff-e87fc223a3fc5310a2dc415999059d623bb89cd10f03324ac4c4496cd7f5419f

Reply:
@delcypher I don't think it has to mention `llvm`. Can we make it even shorter, e.g., `__trap_msg`?

Reply:
I think we usually use some llvm or clang prefix in these sort of things, to reduce the chance of conflict with some other implementation's reserved names? Could be worth a quick grepping for other similar symbol names in clang/llvm to see what we do/if there's some pattern to follow?

Reply:
& not sure it being a compile time constant regex is super important - the regex compilation probably dwarfs the string concatenation required to add the `^` and `:`?

Reply:
I can make it a C++ constant if that's preferable.

Reply:
I don't have any preference, but according to people working on lldb, macro would make it marginally easier to construct the regex in the LLDB frame recognizer.

Reply:
Not a big deal for LLDB either way, whatever people prefer here, we can work around it.